src/mbed_os_tools/detect/platform_database.py

```diff
@@ -170,6 +170,8 @@
         u"0835": u"NUCLEO_F207ZG",
         u"0839": u"NUCLEO_WB55RG",
         u"0840": u"B96B_F446VE",
+        u"0854": u"NUCLEO_L552ZE_Q",
+        u"0855": u"DISCO_L552QE",
```

Comment:
This is currently entered into the online database as `DISCO-L552E`. Should the `Q` be in there?

Reply:
@jeromecoutant I'm okay to amend the database. Let me know of your preference for target name.

Reply:
Could either of you let me know when the database has been changed? Once that's done we can bring this in, thanks.